tests/benchmark_test_inputs/Economic_KPI_C2_E2/csv_elements/storage_1.csv

```diff
@@ -11,4 +11,6 @@ dispatch_price,currency/kWh,NA,0,0
 soc_initial,None or factor,None,NA,NA
 soc_max,factor,1,NA,NA
 soc_min,factor,0.2,NA,NA
+fixed_thermal_losses_relative,factor,0,,
+fixed_thermal_losses_absolute,kWh,0,,
```

Comment:
I hope that we can keep `fixed_thermal_losses_relative` and `fixed_thermal_losses_absolute` optional parameters. If so, please remove the parameters from all other input files, and only leave one benchmark tests for your own feature.